src/views/settings/filters/parts/modify-filter/modify-outgoing-filter-modal.tsx

```diff
@@ -247,6 +253,16 @@ const ModifyOutgoingFilterModal: FC<ComponentProps> = ({
 		modifyOutgoingFilterRules(toSend)
 			.then(() => {
 				setFetchOutgoingFilters(true);
+				// eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
Could you please add a comment for future reference on why this has to be ignored?

Reply:
Once Giuliano's changes are reflected on devel regarding type I'll update this